cmd/cap1188/main.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
```

Comment:
Please move this file into experimental too as it's a mix of smoketest and tool. I can do refactoring afterward. Then it's good for merge.

Reply:
Sounds good. I'm still a little bit confused about the difference between cmd and smoketest, especially because I based mine off of https://github.com/google/periph/blob/master/cmd/apa102/main.go